tests/cmds/test_users.py

```diff
@@ -305,14 +307,14 @@ def add_alias_success(mocker, cli_state):
 
 @pytest.fixture
 def add_alias_limit_failure(mocker, cli_state):
-    cli_state.sdk.detectionlists.add_user_cloud_alias.side_effect = (
+    cli_state.sdk.userriskprofile.add_cloud_aliases.side_effect = (
         Py42CloudAliasLimitExceededError(create_mock_http_error(mocker))
     )
 
 
 @pytest.fixture
 def remove_alias_success(mocker, cli_state):
```

Comment:
There's an `add_alias_success` mock fixture that I think my be unnecessary (and is currently using the detectionlists client`